src/main/java/com/gmail/shablinski93/App.java

```diff
@@ -1,117 +1,52 @@
 package com.gmail.shablinski93;
 
 import com.gmail.shablinski93.model.Dish;
+import com.gmail.shablinski93.service.DishService;
 import com.gmail.shablinski93.service.impl.DishServiceImpl;
 
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
-import java.sql.SQLException;
 import java.util.List;
 
 public class App {
 
+    private static final DishService dishService = new DishServiceImpl();
```

Comment:
Main is static, it is impossible to call a non-static variable. It is also static because it was tied to a class, not to an object.
Another way is using service inside "main" without access modifiers.

Reply:
There is a third way how to manage such case.
Application entry point is just an entry point and there is no any reason why you should use this method for the logic.
It's better to wrap your application start and extract all the logic.

e.g.

```java
public class App {
    public static void main(String[] args) throws IOException {
        App = new App();
        app.run();
    }

    public void run() {
        //all your logic here
    }
}
```
using this approach you could get rid of all static values.